client/src/components/Form/InputDropdown/InputDropdown.tsx

```diff
@@ -10,17 +10,31 @@ import { ReactComponent as ArrowDown } from '../formIcons/arrow-down.svg';
 import { ReactComponent as ArrowUp } from '../formIcons/arrow-up.svg';
 import './InputDropdown.scss';
 
-const InputDropdown = (props) => {
+
+interface InputDropdownProps {
+    dropdownTitle: string,
+    dropdownId: string,
+    inputValueSave:  (id: string, value: string) => void,
+    dropdownVisibility: (s: string) => void,
+    dropdownVisible: boolean,
+    inputFieldAs?: string,
```

Comment:
inputFieldAs is not deconstructed, so I think it is not used. Maybe it is redundant.

Reply:
Fixed.